crowbar_framework/vendor/assets/stylesheets/_eos-branding.scss

```diff
@@ -646,9 +646,9 @@ $alert-success-bg:            $state-success-bg !default;
 $alert-success-text:          $state-success-text !default;
 $alert-success-border:        $state-success-border !default;
 
-$alert-info-bg:               $state-info-bg !default;
-$alert-info-text:             $state-info-text !default;
-$alert-info-border:           $state-info-border !default;
+$alert-info-bg:               #d1ecf1;
+$alert-info-text:             #0c5460;
+$alert-info-border:           #bee5eb;
```

Comment:
Mostly a question for @rsalevsky... these values may actually be better in our branding patch, not in the upstream theme. But we kind of lost the branding branch, it seems (except for the patch in the package). How do we deal with that now?

Reply:
That patch file can be applied to a local branch using `git am` and regenerated with `git format-patch`. Even still because of the way the scss is being built applying it in a patch there does not make sense.

Reply:
@eapodaca sorry that I was not clear. My point is more than the whole EOS branding would actually not be part of the upstream branding, but instead in the SUSE-specific branding (which in the past was done in a different branch) -- and we still have a small patch for the SUSE-specific branding. So the situation is kind of strange, and should probably be resolved eventually... Either we stop having a SUSE-specific theme, or we move the EOS bits to the SUSE-specific theme.